src/parsl-rp/environment.yml

```diff
@@ -7,5 +7,5 @@ dependencies:
   - setuptools
   - wheel
   - pip:
-      - git+https://github.com/AymenFJA/parsl.git@master
+      - parsl
```

Comment:
Is the required version of `parsl` presented in [Conda](https://anaconda.org/conda-forge/parsl)? if yes, then we don't need to keep it under `pip`-section, thus it will be just
```
...
  - wheel
  - parsl
```